dev/source/docs/building-setup-windows-cygwin.rst

```diff
@@ -123,11 +123,13 @@ Set up directories/paths and extra packages in Cygwin
 
 #. open and then close the "Cygwin64 Terminal" application from the desktop or start menu icon.  This creates initialisation files for the user in the Cygwin home directory.
 
-#. open the "Cygwin64 Terminal" application from the desktop or start menu icon.  Enter the following command to install additional Python packages:
+#. open the "Cygwin64 Terminal" application from the desktop or start menu icon.  Enter the following command to set Python 3.6 to be the default Python and then install additional Python packages:
 
 ::
 
-    pip2 install empy pyserial pymavlink
+    ln -s /usr/bin/python3.6 /usr/bin/python
```

Comment:
doesn't python-is-python3 or python-is-python2 exist in cygwin ?

Reply:
> doesn't python-is-python3 or python-is-python2 exist in cygwin ?

Correct. It previously was python-is-python2, but Cygwin has removed that from their installer a few months ago.

Reply:
oh ok ! I just found it on ubuntu20.04 and I found it much simplier that dealing with symlink, that is why I ask if the same thing exist on cygwin.